base/src/main/scala/co/uproot/abandon/Config.scala

```diff
@@ -36,9 +36,10 @@ object SettingsHelper {
     val cliConf = new AbandonCLIConf(args)
     cliConf.verify()
     val configOpt = cliConf.config.toOption
+    val withVersion = cliConf.version.getOrElse(false)
```

Comment:
The default here should be `true` I think.

That is, for normal usage, the version should be shown in the output. It is only the tests that have a problem with it, and hence they should explicitly disable it.